cf_xarray/parametric.py

```diff
@@ -0,0 +1,864 @@
+from abc import ABC, abstractmethod
+from collections.abc import Sequence
+
+import numpy as np
+import xarray as xr
+from xarray import DataArray
+
+ocean_stdname_map = {
```

Comment:
```suggestion
OCEAN_STDNAME_MAP = {
```

Python convention is to make global variables all caps.